src/apps/content-editor/src/app/views/ItemEdit/components/Header/ItemVersioning/ScheduleFlyout/ScheduleFlyout.js

```diff
@@ -108,6 +108,7 @@ export default class ScheduleFlyout extends Component {
         )
       )
       .finally(() => {
+        this.props.onScheduled?.();
```

Comment:
Parent component needs to know if the user is done setting a schedule so it can either redirect them to the newly-created & scheduled item or stay on the same create item page depending on what option they selected on the header actions once done.